src/igvfd/schemas/index_file.json

```diff
@@ -138,11 +138,13 @@
         }
     },
     "fuzzy_searchable_fields": [
+        "aliases",
```

Comment:
In Index File, Assembly and transcriptome_annotation are calculated properties. Could we still include them in this fuzzy search list?

Reply:
Sure, added.